src/olympia/yara/tests/test_admin.py

```diff
@@ -0,0 +1,94 @@
+import json
+
+from django.contrib.admin.sites import AdminSite
+from django.utils.html import format_html
+
+from olympia import amo
+from olympia.amo.tests import (TestCase, addon_factory, user_factory,
+                               version_factory)
+from olympia.amo.urlresolvers import reverse
+from olympia.yara.admin import YaraResultAdmin
+from olympia.yara.models import YaraResult
+
+
+class TestYaraResultAdmin(TestCase):
+    def setUp(self):
+        super().setUp()
+
+        self.user = user_factory()
+        self.grant_permission(self.user, '*:*')
```

Comment:
Modify `src/olympia/constants/permissions.py` to give the `yara.view_yararesult` permission to admins with `ADMIN_ADVANCED`, and use that permission instead of `*:*` (which very few people have, we try not to depend on that)

Reply:
Later we can decide on a more specific permission and add a new group with that, but I think for now advanced admins should be enough. Maybe @wagnerand has other ideas though

Reply:
Wfm.